source/configuration_manual/sieve/pigeonhole_sieve_interpreter.rst

```diff
@@ -86,14 +86,13 @@ Pigeonhole Sieve interpreter:
-`vnd.dovecot.debug <https://raw.githubusercontent.com/dovecot/pigeonhole/master/doc/rfc/spec-bosch-sieve-debug.txt>`_                             supported (v0.3.0+)                                                                                                         no                    Allows logging debug messages
-`vnd.dovecot.environment <https://raw.githubusercontent.com/dovecot/pigeonhole/master/doc/rfc/spec-bosch-sieve-dovecot-environment.txt>`_         supported (v0.4.14+)                                                                                                        no                    Extends the standard "environment" extension with extra items and a variables namespace for direct access
-`vnd.dovecot.execute <https://raw.githubusercontent.com/dovecot/pigeonhole/master/doc/rfc/spec-bosch-sieve-extprograms.txt>`_                     :ref:`Extprograms Plugin <pigeonhole_plugin_extprograms>` (v0.3+)                                                           no                    Implements executing a pre-defined set of external programs with the option to process string data through the external program
-`vnd.dovecot.extdata <http://hg.rename-it.nl/pigeonhole-0.2-sieve-extdata/raw-file/tip/doc/rfc/spec-bosch-sieve-external-data.txt>`_              :ref:`Extdata Plugin <pigeonhole_plugin_extdata>`                                                                           no                    Allows a Sieve script to lookup information from a datasource external to the script
-`vnd.dovecot.filter <https://raw.githubusercontent.com/dovecot/pigeonhole/master/doc/rfc/spec-bosch-sieve-extprograms.txt>`_                      :ref:`Extprograms plugin <pigeonhole_plugin_extprograms>` (v0.3+)                                                           no                    Implements filtering messages through a pre-defined set of external programs
-`vnd.dovecot.pipe <https://raw.githubusercontent.com/dovecot/pigeonhole/master/doc/rfc/spec-bosch-sieve-extprograms.txt>`_                        `Pipe Plugin <https://wiki.dovecot.org/Pigeonhole/Sieve/Plugins/Pipe>`_ (v0.2),                                             no                    Implements piping messages to a pre-defined set of external programs
+`vnd.dovecot.debug <https://raw.githubusercontent.com/dovecot/pigeonhole/release-0.5/doc/rfc/spec-bosch-sieve-debug.txt>`_                             supported (v0.3.0+)                                                                                                         no                    Allows logging debug messages
+`vnd.dovecot.environment <https://raw.githubusercontent.com/dovecot/pigeonhole/release-0.5/doc/rfc/spec-bosch-sieve-dovecot-environment.txt>`_         supported (v0.4.14+)                                                                                                        no                    Extends the standard "environment" extension with extra items and a variables namespace for direct access
```

Comment:
Complains also locally for me:
```
Warning, treated as error:
documentation/source/configuration_manual/sieve/pigeonhole_sieve_interpreter.rst:90:Malformed table.
Text in column margin in table line 4.
```

Reply:
Changed this to use main instead, does it now work?